packages/grafana-ui/src/components/QueryField/QueryField.tsx

```diff
@@ -225,6 +237,7 @@ export class UnThemedQueryField extends React.PureComponent<QueryFieldProps, Que
             readOnly={this.props.disabled}
             onBlur={this.handleBlur}
             onClick={this.props.onClick}
+            onPaste={this.onPaste}
```

Comment:
As @itsmylife said, if we would add a "noop" default value to this callback prop then we could just use it here directly from the props.
```suggestion
            onPaste={this.props.onPaste}
```